js/src/pages/reports/chart-section.js

```diff
@@ -18,13 +18,30 @@ const emptyMessage = __(
 );
 
 /**
- * Renders a report chart.
- *
- * @param {Object} props React props.
- * @param {Array<Metric>} props.metrics Metrics to display.
- * @param {boolean} props.loaded Whether the data have been loaded.
- * @param {Array<IntervalsData>} props.intervals Report's intervals data.
+ * Utility to format Date as YYYY-MM-DD
  */
+function formatDate( date ) {
```

Comment:
@ankitrox Also, let's use the `format` function from the `@wordpress/date` package. See below where `formatDate` is being used.